test/bigtest/tests/circulation-rules-test.js

```diff
@@ -229,7 +229,7 @@ describe('CirculationRules', () => {
     });
   });
 
-  describe('choosing non locating type - material type', () => {
+  describe('choosing non locating criteria type - material type', () => {
```

Comment:
According to the https://github.com/folio-org/mod-circulation/blob/master/doc/circulationrules.md there are the next criteria types: material type, patrons group, loan type, location, institution, campus and library. So in the current test case we selects a criteria type - material type and put its name - m (criteria name) to the editor. 

What about the word "locating", so in this case it means "Any criteria type that is used to identify location (location, institution, campus, library).

Reply:
Of course, thank you for more context). The word `locating` is a kind of improper one here I guess. As I understand this is the gerund of the word 'locate'. I think 'location' word should be used instead. Also, after reading the doc, I think 'criteria' does make sense, however, I found that in the test file it is not used in all places. So, I suggest either add in all places alongside the 'type' word or remove it for consistency.

Reply:
Yes, i tried not to use the word 'location' because there is a criteria type 'location'. So I think, we will have some misunderstanding in case of location instead of locating

Reply:
I checked other places and it should be ok for now

Reply:
I suggest using `location` as the word `locate` has another meaning. We already use a variable like that in the code.

https://github.com/folio-org/ui-circulation/blob/master/src/constants.js#L361

Reply:
Discussed to leave as is.